sstmac/sst_core/integrated_core.h

```diff
@@ -54,7 +54,7 @@ Questions? Contact sst-macro-help@sandia.gov
 
 #include <sstmac/hardware/common/connection.h>
 
-#include <Python.h>
+#include "sst_python.h"
```

Comment:
This needs to be added to the Makefile.am either as an installed header or EXTRA_DIST.

Reply:
Otherwise it doesn't get included in the tarball when doing make dist